design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,361 @@
+type Query {
+
+    getNegotiableQuote(
+        id: ID!
+    ): NegotiableQuote @doc(description:"Get negotibale quote")
+
+    getAllNegotiableQuotes: [NegotiableQuote]! @doc(description: "Get all negotiable quotes")
+}
+
+type Mutation {
+
+    updatedPrices(
+        updatedQuotes: UpdatedPricesInput!
```

Comment:
What is the benefit of having explicit operation to trigger price recalculation? It should probably be done automatically during the appropriate mutations.

Reply:
Since this is a part of NQ REST API we included this. Also here are few use cases (given by Magento).

* Refreshes item prices, taxes, discounts, cart rules in the negotiable quote as per the latest changes in the catalog / shared catalog, and in the price rules. Depending on the negotiable quote state and totals, all or just some of the quote numbers will be recalculated. 

* 'Update Prices' parameter forces refresh on any quote that is not locked for the admin users, including the quotes with a negotiated price.

* The request can be applied to one or more quotes at the same time.

Reply:
@kokoc , as an architect of B2B, please confirm that you are okay with having this mutation